app/views/posts/index.html.erb

```diff
@@ -1 +1,53 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<section class="my-bio bg-secondary bg-gradient">
+    <div class="container">
+        <div class="row">
+        <%= render partial: 'users/user', locals: { user: @user } %>
+        </div>
+        <% @posts.each do |post| %> 
+            <div class="card my-4">
+                <div class="card-header">
+                    <h4># <%= link_to post.Title, user_post_path(@user, post) %></h4>
+                </div>
+                <div class="card-body"><p class="mt-2"><%= truncate(post.text, length: 200, omission: '...') %></p></div>
+                <div class="media-info">
+                    <ul>
+                        <li><a href="#">Comments: <%=post.comments_count%></a></li>
+                        <li><a href="#">Likes: <%=post.likes_count%></a></li>
+                    </ul>
+                </div>
+                <div class="accordion">
+                    <div class="accordion-item">
+                        <h2 class="accordion-header">
+                            <button class="accordion-button" type="button" data-bs-toggle="collapse"
+                                data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                                Comments
+                            </button>
+                        </h2>
+                        <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne">
+                            <div class="card-body">
+                                <!-- Single comment-->
+                                <%if post.recent_comment.count > 0%>
+                                    <% post.recent_comment.each do |comment| %>
+                                        <div class="d-flex mb-4">
```

Comment:
- [ ] The `recent_comment` method cannot be recognized as well. Please make sure you use the right method name